index.html

```diff
@@ -240,13 +241,30 @@ <h2>Contact me</h2>
           needs coding.
           I’d love to help with it!</p>
       </header>
-      <form action="submit">
-        <input type="text" placeholder="Lorem Ipsum">
-        <input type="email" placeholder="loremipsum@gmail.com">
-        <textarea name="texto" rows="4" cols="30">Write your message here...</textarea>
-        <input class="btn" type="submit" value="Get in touch">
+
+      <form action="https://formspree.io/f/mbjezeel" method="post">
+        <ul>
+          <li>
+            <label for="name">Name:</label>
+            <input type="text" id="name" name="user_name" placeholder="Lorem Ipsum">
+          </li>
+
+          <li>
+            <label for="mail">Email:</label>
+            <input type="email" id="mail" name="user_email" placeholder="loremipsum@gmail.com" required>
+          </li>
+
+          <li>
+            <label for="msg">Message:</label>
+            <textarea id="msg" name="user_message" rows="4" cols="30" required>Write your message here...</textarea>
+          </li>
+          <li class="button">
+            <button type="submit">Get in touch</button>
+          </li>
```

Comment:
- [ ] Good job working on the contact form 👍, but you still need to fix some parts make sure you stuck to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) as it is [required](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md#:~:text=You%20must%20stick%20to%20the%20design%20as%20much%20as%20possible%20(e.g.%2C%20font%2C%20colors%2C%20images%2C%20text%2C%20margins)%20using%20the%20templates%20in%20Figma.) 😃
  - Kindly make sure you added your `border-top-left-radius` to your `section`  to look like  [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) as it is [required](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md#:~:text=You%20must%20stick%20to%20the%20design%20as%20much%20as%20possible%20(e.g.%2C%20font%2C%20colors%2C%20images%2C%20text%2C%20margins)%20using%20the%20templates%20in%20Figma.) 😃
  - Kindly make sure you removed the `border`, on the `form` , and the `<label>` tag to look like [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1)
  - Also make sure you `background-image` look like [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) as it is [required](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md#:~:text=You%20must%20stick%20to%20the%20design%20as%20much%20as%20possible%20(e.g.%2C%20font%2C%20colors%2C%20images%2C%20text%2C%20margins)%20using%20the%20templates%20in%20Figma)
  - It could be bettre is you add the `placeholder` to your `textarea` inside of put the value directly , it'll be something like this: 
````
<textarea id="msg" name="user_message" rows="4" cols="30" required placeholder="Write your message here..."></textarea>

````

> | | Figma template | | Your project | |
> |-|---------|-|--------|-|
> | |![Capture](https://user-images.githubusercontent.com/57408419/230066558-6e409f2b-883a-46f7-8f30-d0d798723b9b.JPG)| |![Capture](https://user-images.githubusercontent.com/57408419/230120291-9ee9d7ba-7517-4183-bb88-cb061fad8dcb.JPG)| |

Reply:
@joel-kalema thanks for the review and changes are made!

Reply:
@joel-kalema can't submit to another review because it said that is pending!! can you help me?